Framework/Algorithms/src/FitPeaks.cpp

```diff
@@ -442,6 +444,18 @@ void FitPeaks::init() {
 std::map<std::string, std::string> FitPeaks::validateInputs() {
   map<std::string, std::string> issues;
 
+  // check that min/max spectra indices make sense - only matters if both are specified
+  if (!(isDefault(PropertyNames::START_WKSP_INDEX) && isDefault(PropertyNames::START_WKSP_INDEX))) {
```

Comment:
@peterfpeterson 
I just spotted this while working in this area. I guess this line is supposed to be `isDefault(PropertyNames::START_WKSP_INDEX) && isDefault(PropertyNames::STOP_WKSP_INDEX)` right?